backend/app/crud/__init__.py

```
@@ -25,6 +25,13 @@
     get_content_items,
 )
 
+# Import from crud_ai_conversation.py
+from .crud_ai_conversation import (
+    create_ai_conversation,
```

Ruff (0.11.9)

`.crud_ai_conversation.create_ai_conversation` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)